data/features.yaml

```diff
@@ -300,3 +300,9 @@ features:
       maturity: Alpha
       nextExpectedPromotion: ""
     area: Core
+  - name: "Revision tags (stable revision labels)"
```

Comment:
the docs throught have removed `x experimental`; checking this is correct given what's here?

Reply:
Yes the decision from UX working group was to graduate the command from experimental even though the corresponding feature hasn't been officially promoted yet.